docs/src/transports/aws-cloudwatch-logs.md

```diff
@@ -0,0 +1,193 @@
+It uses the [AWS SDK for JavaScript CloudWatchLogs](https://www.npmjs.com/package/@aws-sdk/client-cloudwatch-logs).
+
+## Installation
+
+::: code-group
+
+```sh [npm]
+npm install loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+```sh [pnpm]
+pnpm add loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+```sh [yarn]
+yarn add loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+:::
+
+## Permissions
+
+The transport uses the [PutLogEvents](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html) action to send the logs to CloudWatch Logs. If you're using the `createIfNotExists` option, it will also uses the [DescribeLogGroups](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogGroups.html), [DescribeLogStreams](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html), [CreateLogGroup](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_CreateLogGroup.html) and [CreateLogStream](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_CreateLogStream.html) actions. Make sure you have the necessary permissions to use them. See more details in the [CloudWatch Logs permissions reference](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/permissions-reference-cwl.html).
+
+## Usage
+
+```typescript
+import { LogLayer } from 'loglayer';
+import { CloudWatchLogsTransport } from "@loglayer/transport-aws-cloudwatch-logs";
+
+// Create LogLayer instance with CloudWatch Logs transport
+const log = new LogLayer({
+  transport: new CloudWatchLogsTransport({
+    groupName: "/loglayer/group",
+    streamName: "loglayer-stream-name",
+  }),
+});
+
+// Use LogLayer as normal
+log.withMetadata({ customField: 'value' }).info('Hello from Lambda!');
+```
+
+The AWS SDK will load your default aws profile or environment. However, you can also specify the cloudwatch client config by yourself via the `clientConfig` option as follows:
+
+```typescript
+import { LogLayer } from 'loglayer';
+import { CloudWatchLogsTransport } from "@loglayer/transport-aws-cloudwatch-logs";
+
+// Create LogLayer instance with CloudWatch Logs transport
+const log = new LogLayer({
+  transport: new CloudWatchLogsTransport({
+    clientConfig: {
+      region: "us-east-1",
+      credentials: {
+        accessKeyId: "<your access key>",
+        secretAccessKey: "<your secret key>"
+      },
+    }
+  })
+});
+```
+
+See the [available options for CloudWatchLogsTransport](#configuration-options).
+
+### Worker thread strategy
+
+By default, each logs entries is sent in the same thread independently. However, you may want to use worker threads via `CloudWatchLogsWorkerQueueStrategy` to improve performance:
```

Comment:
Create a section called "Processing strategies", give a high-level overview of what this is, then create a subsection called "Default strategy" and move the default text to it.